contracts/package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-typescript": "^7.16.0",
+    "@types/node": "^22.5.5",
```

Comment:
This file does not reference o1js directly!  We are getting o1js via o1js-pack....

Reply:
After fixing this locally, my script continues running.

Reply:
I removed it while debugging the wasm TypeError. After adding it back, it works fine on my end as well. I’ll add it in the next commit.